howto/unicode.po

```diff
@@ -154,22 +164,24 @@ msgid ""
 "rules for translating a Unicode string into a sequence of bytes are called a "
 "**character encoding**, or just an **encoding**."
 msgstr ""
-"Pour résumer la section précédente : une chaîne Unicode est une séquence de "
-"points de code, qui sont des nombres de 0 à ``0x10FFFF`` (1 114 111 en "
-"décimal).  Cette séquence doit être représentée sous la forme d'un ensemble "
-"d'octets (c'est à dire, de valeur comprise entre 0 et 255) en mémoire.  Les "
-"règles de traduction d'une chaîne Unicode en une séquence d'octets sont "
-"appelées un **encodage**."
+"Pour résumer la section précédente : une chaîne Unicode est une séquence de "
+"points de code, qui sont des nombres de ``0`` à ``0x10FFFF`` (1 114 111 en "
+"décimal). Cette séquence de points de code doit être représentée en mémoire "
+"sous la forme d'un ensemble de **unités de code**, et les **unités de code** "
+"sont ensuite mappées en octets de 8 bits. Les règles de traduction d'une "
+"chaîne Unicode en une séquence d'octets sont appelées un **encodage de "
+"caractères**, ou simplement un **encodage**."
 
 #: ../Doc/howto/unicode.rst:96
-#, fuzzy
 msgid ""
 "The first encoding you might think of is using 32-bit integers as the code "
 "unit, and then using the CPU's representation of 32-bit integers. In this "
 "representation, the string \"Python\" might look like this:"
 msgstr ""
-"Le premier encodage auquel vous pouvez penser est un tableau d’entiers sur "
-"32 bits. Dans cette représentation, la chaîne « Python » ressemble à ceci:"
+"Le premier encodage auquel vous pourriez penser est l'utilisation d'entiers "
```

Comment:
```suggestion
"Le premier encodage auquel vous pouvez penser est l'utilisation d'entiers "
```

Reply:
OK, mais quelques explications :) ?

Reply:
L'anglais est plus poli que le français. Dans le même genre, nous traduisons « you should » par « vous devez ».

Reply:
merci